R/data.surrogateProfiles.r

```diff
@@ -75,5 +75,7 @@ data.surrogateProfiles = function(data, std=1, ref_data) {
         data_mx_surr=data_mx_surr[-which(var.met == 0),]}
     rownames(data_mx_surr) = tolower(rownames(data_mx_surr))
     data_mx_surr = apply(data_mx_surr, c(1, 2), as.numeric)
-    return(data_mx_surr)
+    
+    # add as.data.frame for calling from Python
+    return(as.data.frame(data_mx_surr))
```

Comment:
Will this brake R execution?

Reply:
It is possible, please ignore all changes made to the R code (except the ranks saving in the main CTD script).